src/docker-pest-command.js

```diff
@@ -3,9 +3,13 @@ const RemotePestCommand = require('./remote-pest-command');
 
 module.exports = class DockerPestCommand extends RemotePestCommand {
     get paths() {
-        let paths = {}; 
-        for (const [localPath, remotePath] of Object.entries(this.config.get("docker.paths"))) {
-            paths[ localPath.replace('${workspaceFolder}', vscode.workspace.rootPath) ] = remotePath; 
+        let paths = {};
+        const workspaceFolder = vscode.workspace.workspaceFolders ? vscode.workspace.workspaceFolders[0].uri.fsPath : null;
```

Comment:
This can be improved even further to support multiple workspaces. If `workspace.folders.length > 1`, then use `vscode.window.showWorkspaceFolderPick` to let the user pick one, then save the choice in a global variable (_not persisted_) to avoid asking to choose every time.